src/code.rs

```diff
@@ -806,9 +799,10 @@ fn build_default_impl_fn(struct_name: &str, columns: &Vec<ParsedColumnMacro>) ->
         })
         .collect::<Vec<String>>()
         .join(",\n");
-    buffer.push_str(fields_to_defaults.as_str());
-    buffer.push_str("\n        }\n    }\n}");
-    buffer
+    format!(
+        "impl Default for {struct_name} {{\n    fn default() -> Self {{\n        Self {{\n{f2d}\n        }}\n    }}\n}}",
+        struct_name = struct_name, f2d=fields_to_defaults.as_str()
```

Comment:
```suggestion
        f2d=fields_to_defaults.as_str()
```

unless i missed something, that shouldnt be required anymore.

i think `f2d` should also be able to be directly embedded.

Reply:
Ok fixed